wist-scoreboard-generator/src/app/Pages/homepage/homepage.component.scss

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
```

Comment:
is this needed if you import it in styles.scss?

Reply:
There where two different weights for the fonts, the one from styles.css was unused, so I moved this one there and deleted the one that was too thick